### ISSUE TITLE
Configurable product page broken with this module

### ISSUE THREAD

**Issue by clivewalkden** [Smile-SA/magento2-module-debug-toolbar#1]
Hi,

Thanks for creating a great plugin! 

I've noticed that a configurable product fails to load in the admin with your module enabled. In my setup your JS scripts are inserted before the Magento_Backend/js/bootstrap/editor.js script and cause requirejs to fail with the following:

```
Uncaught Error: Mismatched anonymous define() module: function(){return t.hljs}
http://requirejs.org/docs/errors.html#mismatch
    at makeError (require.js:166)
    at intakeDefines (require.js:1221)
    at Object.localRequire [as require] (require.js:1402)
    at requirejs (require.js:1737)
    at editor.js:7
```

This then makes the `dynamic-rows-configurable.js` script to throw several errors and stop the page rendering.

```
Uncaught TypeError: this.getChildItems(...).each is not a function
    at UiClass.processingUnionInsertData (dynamic-rows-configurable.js:232)
    at setNested (objects.js:43)
    at Object.nested (objects.js:117)
    at UiClass.set (element.js:305)
    at updateValue (links.js:80)
    at Function.notifySubscribers (knockout.js:1103)
    at Function.observable.valueHasMutated (knockout.js:1300)
    at UiClass.observable [as unionInsertData] (knockout.js:1285)
    at UiClass.initChildren (dynamic-rows-configurable.js:105)
    at UiClass.initialize (dynamic-rows.js:191)
```
```
Uncaught TypeError: max.each is not a function
    at UiClass._checkGridData (dynamic-rows-grid.js:269)
    at UiClass.processingInsertData (dynamic-rows-grid.js:207)
    at setNested (objects.js:43)
    at Object.nested (objects.js:117)
    at UiClass.set (element.js:305)
    at updateValue (links.js:80)
    at Function.notifySubscribers (knockout.js:1103)
    at Function.observable.valueHasMutated (knockout.js:1300)
    at UiClass.observable [as insertData] (knockout.js:1285)
    at setNested (objects.js:43)
```
```
Uncaught TypeError: max.each is not a function
    at UiClass._checkGridData (dynamic-rows-grid.js:269)
    at UiClass.processingInsertData (dynamic-rows-grid.js:207)
    at setNested (objects.js:43)
    at Object.nested (objects.js:117)
    at UiClass.set (element.js:305)
    at updateValue (links.js:80)
    at Function.notifySubscribers (knockout.js:1103)
    at Function.observable.valueHasMutated (knockout.js:1300)
    at UiClass.observable [as insertData] (knockout.js:1285)
    at setNested (objects.js:43)
```

With your module turned of for the admin it still throws the error. When you use the CLI to disable the module the page then loads again as expected.

Kind regards,

Clive

**Comment by spipu** [comment]
thanks, i will check this

**Comment by spipu** [comment]
what is you magento version / edition ?

**Comment by spipu** [comment]
fixed in the new release 3.1.1

**Comment by clivewalkden** [comment]
Sorry, that was version 2.2.5 of Magento
